datashare-cli/src/main/java/org/icij/datashare/cli/DatashareCliOptions.java

```diff
@@ -119,13 +119,15 @@ public final class DatashareCliOptions {
     public static final String TCP_LISTEN_PORT_OPT = "tcpListenPort";
     public static final String VERSION_ABBR_OPT = "v";
     public static final String VERSION_OPT = "version";
+    public static final String ARTIFACT_DIR_OPT = "artifactDir";
 
     private static final Path DEFAULT_DATASHARE_HOME = Paths.get(System.getProperty("user.home"), ".local/share/datashare");
     private static final Integer DEFAULT_NLP_PARALLELISM = 1;
     private static final Integer DEFAULT_PARALLELISM = Runtime.getRuntime().availableProcessors() == 1 ? 2 : Runtime.getRuntime().availableProcessors();
     private static final Integer DEFAULT_PARSER_PARALLELISM = 1;
     public static final DigestAlgorithm DEFAULT_DIGEST_METHOD = DigestAlgorithm.SHA_384;
     public static final String DEFAULT_DATA_DIR = Paths.get(System.getProperty("user.home")).resolve("Datashare").toString();
+    public static final String DEFAULT_ARTIFACT_DIR = DEFAULT_DATA_DIR;
```

Comment:
I think we should avoid mixing the artifact dir and the data dir. I would create a dedicated directory in the Datashare's home:

```suggestion
    public static final String DEFAULT_ARTIFACT_DIR = DEFAULT_DATASHARE_HOME.resolve("artifact").toString();
```

Reply:
yes I removed the default value from the option, but I forgot this.
Thank you.